src/migrations/20221220160345-user-pat-permissions.js

```diff
@@ -0,0 +1,33 @@
+exports.up = function (db, cb) {
+    db.runSql(
+        `
+        INSERT INTO permissions (permission, display_name, type) VALUES ('READ_USER_PAT', 'Read PATs for a specific user', 'root');
+        INSERT INTO permissions (permission, display_name, type) VALUES ('CREATE_USER_PAT', 'Create a PAT for a specific user', 'root');
+        INSERT INTO permissions (permission, display_name, type) VALUES ('DELETE_USER_PAT', 'Delete a PAT for a specific user', 'root');
+
+        INSERT INTO role_permission (role_id, permission_id)
```

Comment:
Does admin need entry in role_permission? I thought they had access to everything automatically. Just wondering.

Reply:
You are right @sjaanus, admin is special and we do not need to map these permission for that role.

Reply:
Thanks, addressed in https://github.com/Unleash/unleash/pull/2718/commits/b379484102dabb8d3826535e5cc9fe1e9b3cb8c7